imports/wasi_snapshot_preview1/fs.go

```diff
@@ -382,7 +382,7 @@ func getWasiFiletype(fm fs.FileMode) uint8 {
 	}
 }
 
-func writeFilestat(buf []byte, st *platform.Stat_t) (err error) {
+func writeFilestat(buf []byte, st platform.Stat_t) (err error) {
```

Comment:
I think in this context it would be OK to keep it a pointer because the compiler sees through the function and knows it isn't being retained.

It's also good to keep usage consistent and pass it by value everywhere 👍

Reply:
cool I'll change this one. thanks!